packages/select-rich/test/lion-listbox-invoker.test.js

```diff
@@ -0,0 +1,76 @@
+import { expect, fixture, html, defineCE } from '@open-wc/testing';
+import { LionButton } from '@lion/button';
+import { LionListboxInvoker } from '../src/LionListboxInvoker.js';
+
+import '../lion-listbox-invoker.js';
+
+describe('lion-listbox-invoker', () => {
+  it('should behave as a button', async () => {
+    const el = await fixture(html`
+      <lion-listbox-invoker></lion-listbox-invoker>
+    `);
+    expect(el instanceof LionButton).to.be.true;
+  });
+
+  it('should render value based on selectedElement', async () => {
+    const el = await fixture(html`
+      <lion-listbox-invoker></lion-listbox-invoker>
+    `);
+    expect(el).lightDom.to.equal('');
+    const myNode = document.createElement('div');
+    myNode.textContent = 'foo';
+    el.selectedElement = myNode;
+    await el.updateComplete;
+    expect(el).lightDom.to.equal('foo');
+  });
+
+  it.skip('should render multiple values based on selectedElements', async () => {});
+
+  describe('Subclassers', () => {
```

Comment:
Am I correct to assume this will eventually either be removed from the test (if we decide not to implement a subclass ourselves) or transformed into a multi-select test? For early BDD I guess this makes sense for now :)

Reply:
It is mainly meant to show the intended use of our lion layer, regardless of multiselect. See https://github.com/ing-bank/lion/blob/feat/richSelect/packages/select-rich/docs/DesignConsiderations.md for more context